### ISSUE TITLE
25.道路：起始步骤，准备工作和决定

### ISSUE THREAD

**Issue by Pathwork-In-China** [Pathwork-In-China/Pathwork-Lectures#27]
25. The Path: Initial Steps, Preparation, and Decisions
### 道路：起始步骤，准备工作和决定
Pathwork Guide Lecture No. 25 | March 14, 1958
心路指导讲座 第25篇 | 1958.3.14

25.1  Greetings in the name of God and Jesus Christ. I bring blessings for all of you, my friends. From all my preceding lectures you will have understood one thing clearly:  the necessity for self-development on this earth plane which exists for that very purpose. No matter how difficult life may be at times, only those who fulfill this purpose can find peace in their souls. I have promised to start this course on development so that each one of you can find your way by learning how to go about it, where to begin, and what is involved in doing the work. Moreover, I want to tell you before I begin that many of the words I shall give you in these particular lectures can be treated as a meditation. You should retain these words and not just read them once, for that will not be enough. You should meditate on these teachings, so that this knowledge may eventually grow from being superficial and intellectual to reaching the deeper regions of your being. Only then will they be really beneficial to you.
以上帝和耶稣基督之名问候你们。我为你们所有人带来祝福，我的朋友们。从所有我之前的讲座中，你们应该已经非常清楚地了解了一件事：（那就是）在这个地球位面——它存在的意义就在于此——上进行自我发展的必要性。不论生活在有些时候变得多么困难，只有那些达到这个目的的人才能在灵魂中找到安宁。我已经承诺要开始这个关于（如何）发展的课程，从而使你们每个人都能通过学习怎样进行（这种自我发展）、从哪里开始、这份劳动中涉及到哪些东西，来找到自己的路。另外，在开始之前我先告诉你们，这些我将在此系列特别的讲座中给出的话可以被当成冥想（的材料）。你们应该把这些话留在心中而不是只读一遍，因为那是不够的。你们应该对这些教导进行冥想，使这份知识最终能从肤浅的智力层面知识开始进行生长，并触及到你们身心的更深层区域。只有那时，它们才会真正对你有益。

25.2  Everybody knows that it is important to be a decent person, not to commit so-called sins, to give love, to have faith, and to be kind to others. However, this is not enough. In the first place, knowing all this and actually being able to act on it are two different stories. You may be able by voluntary action to refrain from committing a crime such as stealing or killing, but you cannot possibly force yourself to feel that you do not want to harm anybody, ever. You may act kindly toward another, but you cannot force yourself to feel kindly. Neither can you force yourself to have love in your heart or to have real faith in God. Whatever pertains to emotions is not dependent upon your direct actions or even on your thoughts. Changing your feelings requires the slow process of self-development and self-recognition.
每个人都知道重要的是，去成为一个体面的人，不要犯所谓的（宗教意义上的）罪，要去给予爱，要有信念，还要对别人友善。然而，这是不够的。根本来说，“知道这一切”和“能够实际践行它们”是两回事。你也许能通过自发行为对某种犯罪行为（比如偷窃或谋杀）进行克制，但你永远不可能强迫自己产生“我不想伤害任何人”的感觉。你可以对另一个人表现得友善，但你没法强迫自己产生友善的感觉。同样你也不能强迫自己在心中产生爱的感觉，或者产生对上帝真正的信念。从属于情感和情绪范畴的东西是不依赖于你的直接（外部）行为、甚至是你的思想的。要改变你的感觉，这需要进行缓慢地自我发展和自我认知过程。

25.3  You may realize that you do not have enough faith, but realizing this and trying to force yourself to have it by telling yourself, “I must have faith,” will not bring you one step closer:  quite the contrary. Superficially you may be able to talk yourself into it, but this does not mean your faith or your ability to love is real. It is already a great accomplishment for people of lower spiritual development if they can be brought to not committing wrong deeds, but this is certainly not sufficient for any one of you here. More is expected of you.
你可能意识到自己没有足够的信念，但认识到这一点并试图通过告诉自己“我必须拥有信念”来强迫自己产生它，是不会有效果的：事实上会产生相反的效果。表面上你也许能说服自己，但这不意味着你的信念或者你去爱的能力是真实的。对精神发展程度较低的人们来说，如果他们能不做出错误的行为举动的话，这已经是很大的成就了，但是这对你们在这里的任何一个人来说都是不够的。你们身上带有的期望更高。

25.4  Primarily this path is about changing your innermost feelings. How to go about that is the question that we have to begin with. There I can show you the way. In the first place, my friends, you cannot change anything as long as you do not know what is really in you. The greatest difficulty on this path is that people tend to fool themselves about who they really are. Now I am not only talking about the subconscious mind which you all know exists. I will not even go as far as that. Between the conscious and the subconscious mind there is another layer which is much closer to the conscious mind. However, you are still unaware of this layer because you want to be unaware. You escape from it although its symptoms and signs may be right under your nose. People flee from such awareness because they mistakenly think that what they do not know does not exist. You may not think so in exactly these words, but feelings of this sort go on in you without your quite realizing them. However, these hidden feelings exist. Even if you turn away from your own inner reality—your temporary reality of this present time—this layer is still part of the reality of your life and your present state of development.
根本上来说，这条道路是关于改变你们的内心最深处的感觉的。如何着手处理这一点，就是我们最开始要考虑的问题。这里我可以为你们指路。首先来说，我的朋友们，当你不知道自己内心有什么的时候，你就没法改变任何东西。这条道路上最大的难度在于，人们倾向于在“自己到底是什么样的人”这一点上欺骗自己。这里我指的不是大家熟知的潜意识头脑。我甚至不会说的那么远。在清醒/显意识和潜意识头脑之间，还有一层意识，它离显意识头脑近得多。然而，你们对这个层次仍然没有觉察，因为你们想要保持无所察觉的状态。你们逃离它，尽管它的症状和迹象可能就在你们的鼻子底下。人们逃离这种觉察，这是因为他们错误地认为：自己不知道的东西就不存在。也许你们没有用这种完全一致的话语在思考，但这类型的感觉却在你们心中运行着，而你们没怎么意识到这一点。尽管如此，这些隐藏的感觉是存在的。就算你把注意力从自己的内心现实——也就是当前时刻的临时现实——上转开，这一层意识仍然是你的生活现实和你目前发展状态的一部分。

25.5  You may all remember the lecture I gave some time ago about the higher self, the lower self, and the mask self. What I have explained is part of the mask self, but it is not only a mask. I may call it the exterior self which does not always correspond to the inner person. All of you know that it is wrong to do or think or feel certain things. If these feelings still exist in your lower self, you turn away from them, thinking you have thereby eliminated what you recognize as wrong. Avoidance or denial is the greatest mistake a human being can make, for it causes infinitely more trouble, more problems, and more inner and outer conflicts than anything you know in your conscious mind.
你们也许都记得我一段时间前做的关于高级自我、低级自我和面具自我的讲座。我刚才解释的是属于面具自我的一部分，但它不仅仅是一个面具。我可以称它为“外表自我”，它不总是与内部人格相对应。你们都知道去做、去思考或者去感觉某些事是不对的。如果这些感觉仍然存在于你的低级自我中，你却无视他们，相信这样你就把自己认为是错的东西排除掉了。回避和否认是一个人能犯的最大错误，因为它相比任何你们显表头脑所知道的东西来说，会引起无法计数的更多矛盾、更多问题以及更多内部和外部冲突。

25.6  I have often mentioned the various spiritual laws which are constantly being violated by human beings. The process I have just described violates one of these laws. It is the law of facing life. To face life’s reality means to be able to face yourself as you are with all your imperfections. If you do not face life first, you can never develop. No system trying to teach ways to jump over this hurdle can ever be really successful, for seeking such shortcuts violates a spiritual law.
我经常提到许多持续性地被人类违反的精神法则。我刚刚描述的过程就违反了这些法则中的一个。这就是“面对生活”的法则。面对生活的现实意味着，（一个人）能够实事求是地面对他自己，包括自己所有的不完美之处。如果你不首先面对生活，你就永远没法发展。任何试图教授跳过这道障碍的方法的系统都永远没法真正成功，因为寻求这种捷径的做法违反了精神法则。

25.7  All of you are unconsciously following this harmful process all the time, even though some of you may have already gained a certain amount of self-knowledge. There is not a single one of you who has not had at least one realization about an inner trend that has made this trend really conscious. Nevertheless, in many other areas your conscious mind still flees from facing the inner truth. You may even know your shortcomings, but you certainly do not know all your real motives. You do not understand why you have certain opinions, tastes, or idiosyncrasies; even your good qualities may be partly influenced by an unconscious fault or wrong inner current. These trends and tendencies about which you have hitherto deceived yourself have to be understood with respect to the influences and connections they have.
你们所有人一直都在无意识地进行这个有害的过程，尽管你们有些人已经获得了一定数量的自我知识。你们当中没有谁还从来没有发现至少一个这样的内部趋势，并且把它认识清楚。尽管如此，在许多其他方面，你们的显意识头脑仍然逃避面对这一类的内部现实。你们甚至可能知道自己的短处，但你当然不会知道自己心中所有的真实动机。你不理解自己为什么会有某种意见、口味或者癖好；就连那些好的品质也可能部分受到了一个无意识的过错或者错误涌流的影响。这些你目前为止都在以此欺骗自己的趋势和倾向必须得到正确的理解，包括它们的影响和产生的联系。

25.8  There is nothing in the human soul that comes entirely from the higher self or the lower self, because everything mixes constantly. Purification means to separate, understand, and rearrange in conscious understanding all these various trends, thus purifying the basic good trends from all masks of self-deception and from influences caused by character weaknesses. The higher self in you says, “I want to be perfect. I know this is the will of God.”  But it is the ignorance of the lower self that makes you think perfection can be attained by turning away from your imperfections and disregarding them. It is also the lower self that always wants to have everything so comfortable. The lower self wants to be in a high position, but for different reasons than the higher self. Your higher self seeks to advance for the love of God by way of recognition and enlightenment, aware that only when you are perfect will you be truly capable of loving your fellow creatures. But your lower self wants to be perfect in order to have more ego-gratification and to swell its head, to be admired. All of you, without exception, also feel this way. Here is an example where both the higher and the lower self want the same thing, but their motives are entirely different. It is of utmost importance for the purification of your personality and for the sake of a healthy and harmonious soul to separate these motives and to recognize their voices. Do not feel I am blaming you, nor should you blame yourself when you begin to recognize these trends in you. I am stating a fact, and one of the basic requirements for your path is that you accept the fact of many negative trends still existing in yourself. Only from this premise can you go on and change the impurity of your motives.
人类灵魂中没有什么东西是完全来自高级自我或低级自我的，因为一切都在持续地混合着。净化意味着以清醒、有意识的理解力对所有这些不同的趋势进行分离、梳理和重组，以此把基本的、好的趋势与所有自我欺骗的面具和由性格弱点引发的影响力分离开，从而实现净化。你内心中的高级自我说，“我想变得完美。我知道这是上帝的意志。”但是出自低级自我的无知，你认为完美和完善状态可以通过“无视自己的不完美之处”的做法达到。同理，也是低级自我总是在想以舒适的方式处理所有事物。低级自我想要处于一个很高的位置上，但是却出于与高级自我不同的原因。你的高级自我寻求的是，通过认知和启迪的方式推进上帝的爱，并且它知道只有当你完美的时候，你才真正有能力去爱自己的生物伙伴们。但你的低级自我想变得完美，是为了拥有更多的自我满足并使自己的脑袋膨胀、去得到别人的仰慕。你们所有人，没有例外地，也会有这种感觉。这就是高级自我和低级自我同时想要同一件事的例子，但它们的动机是截然不同的。就人格的净化和获得灵魂的健康与和谐来说，把这些动机区分开并识别出它们的声音是极为重要的。当你开始识别自己内心中这些趋势时，不要觉得我在责备你们，也不要责备你们自己。我只是陈述一个事实，而你们进行这条道路的几个基本要求之一就要去接受这个“自己内心仍然存在许多负面趋势”的事实。只有从这个前提出发，你们才能取得进展并且改变自己动机中的杂质。

25.9  You must also recognize the reasons why your lower self turns you away from facing yourself. One reason is that to recognize yourself as imperfect is unpleasant. The other is that the lower self is lazy and never wants to work. Yet it requires work to face what is in you, especially when it comes to facing the unpleasant things. So the first step, my friends, in your decision to walk the path of self-development and purification is to become clear about this. It is the greatest decision a human being can ever make. It is the noblest of all fights humanity can ever engage in. But you must do so with open eyes; therefore realize what you ought to expect and what you have to search for. Do not start to search for perfection immediately. This again would be unrealistic because you cannot attain perfection quickly without having spent much time and effort. This search is the first and most important half of the truth of the path. If you proceed well on this part of the path, you have won half the battle already, and even more, my friends!  If you realize this, you will not be discouraged when you are busy on this first half of the work that is necessary. You can reach perfection only by going through your imperfections, not by going around them. You should meditate on this daily until such knowledge becomes part of you and until you are imbued with this truth.
你也必须认清“你的低级自我使你远离面对自己”的原因。其中一个原因是“看到自己是不完美的这件事会令自己不愉快”。另一个原因是，低级自我很懒而且从来不愿意干活。然而，要面对自己内心存在的东西是需要努力的，特别是在面对令人不愉快的事情时更是这样。所以你们走上自我发展和自我净化道路的第一步，我的朋友们，就是要认清这一点。这是一个人所能做出的最重大的决定。这是全人类所能参与的最高贵的斗争。但你们必须睁大眼睛来这样做；因此，要意识到自己会遇到什么情况，以及自己必须寻找什么东西。不要立马开始寻找完美状态。这会是不现实的，因为在没有花费足够的时间和努力之前，你是没法达到完美的状态的。这种搜寻是这条道路上第一个，也是最重要的一个真理。如果你们在道路的这一部分上进展顺利的话，就已经赢得了战斗的一半，甚至更多，我的朋友们！如果你意识到这一点，那么当你忙于处理这份工作中必要的第一部分时，就不会感到气馁。你只有通过处理自己的不完美之处才能变得完美，而不是绕过它们。你们应该每天都在这一点上进行冥想，直到这份知识变成你的一部分，直到你被这份真理所充满。

25.10  Furthermore, prepare yourself that going on this path does not mean a constant and smooth improvement of yourself and your life conditions. This again is completely unrealistic. It is necessary for you to face the fact that the path is long and that repercussions and times of testing will not cease as quickly as you would like to believe. I might say at this point that a great deal of harm is being done when people are led to understand that by following certain rules of metaphysical teachings their problems will altogether cease, or that if they appear to cease for a time, that will prove a sign of success. There are many people who appear to have no outer problems whatsoever. Such people are certainly not on this path. They may be entities of lower development of whom less is expected in this incarnation and who have been given a chance to prove what they can do with an easy life. If they do not do their best now, they will have more difficult times in a future incarnation, perhaps so they can prove themselves. But to imagine that going on this path of purification will immediately lessen your troubles or problems is very immature and childish. Certainly your outer and inner problems will lessen and finally cease, but only after a long time, after you have first thoroughly understood your inner makeup and rearranged your inner currents. In this way you will dissolve inner images that are directly responsible for your conflicts. Once you have attained some victories over yourself you will fully realize this truth, but it will take a long time and years of work. Then very gradually the times of trial will lessen in impact and frequency, as harmony grows in your soul and as you take charge of yourself and become aware of who you are. When I say “aware of yourself,” I mean knowing your lower self utterly and completely, which does not have to mean that you have overcome it completely.
进一步来说，要让自己意识到，走上这个道路并不意味着一段对自己和自己生活条件的持续的、平滑的改进。这也是完全不现实的。你们有必要面对这样一个事实：这条道路很长，而其中的反弹、反复和试炼的时间不会如你所期望的那么快地消失。这里我甚至可以说，当人们被引导着相信“通过遵循某种形而上学规则的教导，他们的问题会一股脑儿地消失”，或者“如果这些问题会消停一段时间，这就是取得成功的象征”这类说法时，这会造成很大的危害。有许多人似乎没有任何形式的外部困扰。这些人当然不处在这条道路上。他们可能是处在较低发展阶段的实体，（上帝）对他们当前降世的期望会低一些，而他们获得了证明“自己在舒适的生活中能有怎样表现”的机会。如果他们现在不做到最好，就会在未来的降世中遇到更多困难时刻，也许这样他们更能证明自己。但是，如果以为走上这条（自我）净化的道路会立刻减轻自己的毛病和问题的话，就是很不成熟、很幼稚的了。你的外部和内部问题当然会减轻并且最终停止，但这经常只有在很长一段时间后，在你首先彻底理解了自己的内部组成，并重新布置了自己的内部涌流后才会发生。以这种方式，你会化解掉存在于内心的图像，正是这些图像直接导致了你产生的冲突。当你取得了一些对抗自己的胜利时，你就会完全意识到这个事实，但这需要很长的时间和数年的努力。之后慢慢地，试炼的次数、冲击力和频率会减少，同时你灵魂中的和谐会增加，而你也更能掌控自己并且更加了解自己是谁。当我说“觉察到你自己”时，我指的是彻底、完整地了解你的低级自我，但这不一定意味着你已经完全克服它了。

25.11  When you start on this path, my friends, you ought to meditate every day before you do anything further. Meditate on the fact that you can expect to find aspects of yourself you may be shocked about. Expect this and meet it halfway instead of hiding and escaping from it. Expect that just as you have had tests before you even started on this direct path, tests will still come your way for quite a time. The only difference is that a person who is on the path will, after some successful work, understand that each testing and each time of woe means something very specific. A particular message that there is something quite special to learn about the self is conveyed by each difficult period and every hardship. Only after a considerable time will your mind be trained in this direction so that you will find out faster and faster what the lesson is. The moment you will understand the significance of these periods, this particular testing will cease. As long as you have not understood it, the testing will be there. It may recede after a while, but it will come back in the same or similar form until you have learned the lesson. Those who have experienced what it means to understand the message of a particular hardship, to really understand it to its core, will realize what a blessing it is!  Then what I am saying here will not merely be words, but a deep experience.
当你开始走上这条道路时，我的朋友们，在进一步做任何事之前，你应该每天进行冥想。冥想的内容是“可以预料的是，你会发现自己内心的一些侧面，它们可能会使你感到震惊”这个事实。对此建立合理的预期并主动应对这一点，而不要躲藏起来逃避它。同样可以预料的是，就如你开始这条直接道路之前会遇到试炼一样，在相当一段时间里你仍然会遇到试炼。唯一的区别在于，走在这条道路上的人，在做出一些成功的努力后将认识到，每次试炼和每个苦恼的时刻都包含了某种非常具体的意义。每个苦难时期和每种艰难情况都传递着一条特别的信息，那就是：在所有这些不同的情况，都中有某种关于自我的很特别的东西需要学习。只有在相当长的一段时间后，你的头脑才能在这个方向上得到足够的训练，使你能越来越快地发现（每个情形中）包含的教训是什么。在你能领会这些时期的重要性的那个瞬间，这个特定的试炼就会停止。只要你还没有理解它，这份试炼就会保持存在。它可能会退去一段时间，但（过一会）它又会以相同或相似的形式重新出现，直到你学到这个教训为止。那些体验过“理解某个艰难境况蕴含的信息”，并真正领会了它的核心的人们会知道这是一个多么大的祝福！那时候，我在这的描述就不再仅仅是词语了，而会是一份深刻的体验。

25.12  Meanwhile, the person who is not on this path or who has perhaps not completely found his or her way to the path and is still in the first stages, will be at a loss. Not knowing why he or she has to go through all this makes the hardships infinitely more difficult to bear. Before you can come to the point in your development when testing periods and times of trouble will cease, you must first go through the stage of completely understanding these periods of difficulty, and then meet them in a spirit of courage and wisdom. When you can do all this, the transition period will begin and outer conflicts and problems will no longer touch you to the quick. You will remain quite calm and serene inside during outer hardships. Only after this condition has been reached can hardship begin to cease altogether. You ought to be clear about the timing and the stages. So prepare yourself that these trials will not cease. Your life will outwardly continue for a while as before, until you learn from it that which is so necessary for you to know. If you expect it to be this way and if you go on the path in this frame of mind, you will not be disappointed.
与此同时，那些没有走在这条道路上，或者还没有完全找到进入这条道路的入口并且仍然处于起始阶段的人们，会很茫然。他们内心那种“不知道为什么自己必须经历所有这些”的困惑会使自己所处的困境变得无数倍地更难以承受。在你个人发展中发生的试炼阶段和各种问题能够停止之前，你必须首先对这些困难时段（和它们的含义）进行完全、彻底的理解，并且，以一种勇敢和智慧的精神面对它们。当你能做到这一切时，之后的“过渡阶段”就会开启，此时外部冲突和矛盾就不会再（像之前那样）触到你的痛处了。当你处于外部艰难条件之中时，仍将保持内心平静而安宁的状态。只有达到这个条件后，艰难情境才会开始完全停止。你们应该了解清楚这些时间点和不同阶段。要做好准备，因为这些（生活中的）试炼不会（立刻）停止。直到你从中学到必要的东西之前，你日常生活的外界表现会如之前一样继续一段时间。如果你提前预料到了这一点，并且在这个思想框架内走上这条道路，你将不会（因为预期过高而）感到失望。

25.13  If you go on it like a blindfolded child, full of wishful thinking, then you will be disappointed—not only in God and in what you have somehow unconsciously expected from Him when you chose to follow this path, but also disappointed in yourself and in your endeavors. This path is not a fairy tale; it is reality of the crassest sort, my friends. It is utter reality. Reality is not only hard and difficult and dark, but is also more beautiful than anything else. The beauty of reality is beyond all comparison superior to the puny beauty the escapist imagination concocts to get away from unpleasant reality. Remember that!
如果你像一个蒙住双眼的小孩一样走上这条道路，内心充满了痴心妄想，那你一定会感到失望——不仅是对上帝、对当你选择这条道路时无意中对祂抱有的期望感到失望，而且也会对自己和自己付出的努力感到失望。这条道路不是一个童话故事；它是最粗钝的现实，我的朋友们。它是彻底的现实。现实不仅又坚硬又困难又黑暗，还比任何其他东西都美。现实的美无与伦比地优于逃避现实者为了逃离不讨人喜欢的现实而用想象力调制出来的微不足道的美。记住这一点！

25.14  Another thought for meditation:  when you go on this path, you must also prepare yourself to abide by another of the spiritual laws, which says that there is a price to be paid for everything. Whoever tries to avoid this will finally pay a much heavier price. Every single person is doing this constantly in one way or another; some do it more obviously, others more subtly and secretively. Many people are not doing it outwardly, but psychologically you are all doing it, particularly when you approach this path with only half-open eyes. Realize that there is a price, but the price is well worth it!  When you buy a house and you want a beautiful mansion, you are reconciled to paying an adequate price. You will not expect a mansion or a palace for the price of a shack. On the material level, you have no quarrel with this truth, but on the emotional, psychological, and spiritual levels you wish for a palace for the price of a hut—and sometimes you do not want to pay any price at all.
另一个用来冥想的想法：当走上这条道路时，你们必须准备好遵守另一条精神法则，那就是：“任何事都有相应的代价”。任何试图避免这一点的人，最终将付出一份沉重得多的代价。所有人，无一例外地都一直在各种各样的方式这样做；有人做的明显些，其他人（做的）微妙些、隐秘些。许多人没有外向地这样做，但你们全都在心理层面上这样做，特别是当你们以半睁半闭的眼睛来尝试这条道路时更是这样。意识到其中有一份代价，但这份代价是非常值得的！如果你买房子时，想要买一栋漂亮的宅邸公馆，你会心甘情愿地为其支付一笔可观的费用。你不会期望一座公馆或宫殿只有一间窝棚的价格。在物质层面，你对这个事实没有意见，但在情绪层面、心理层面和精神层面，你想要以棚屋的价格买到一座宫殿——有时你甚至不想支付任何代价。

25.15  That is part of your sick soul. The price you pay by going on this path of development is certainly a high one, but there is absolutely no other means on earth or in heaven to gain harmony, love, happiness, and complete inner security where nothing evil can ever touch you or throw you off balance. The price is:  no self-pity, no self-delusion, utter severance with the little ego, time, effort, patience, perseverance, and courage. What you will receive for this price is indeed a hundred times worth it, but do not expect to see the reward right after you start. By start I mean a period of at least two years of work in this manner, provided you do not work half-heartedly. In other words, and speaking symbolically, your money must first be paid in full.
这种心态是你病态的灵魂的一部分。你能走上这条发展的道路，这本身当然就是一份高昂的代价，但不论是地球上还是天堂里，绝对没有任何其他的方法可以使你获得和谐、爱、幸福快乐和完整的内心安全感（这些东西使得任何邪恶事物都无法触及到你或者使你失去平衡）。其中的代价是：抛弃自怜自艾、抛弃自我欺骗、与“小我”的彻底断绝、时间、努力、耐心、坚持和勇气。你以这些代价换来的收获确实会是百倍的值得，但不要期望自己开始后立马就能看到这些奖励。我这里说的“开始”指的是以这种方式工作至少两年的时间，前提是你没有虚心假意。换句话说，并且象征性地说，你必须一开始就全额付款。

25.16  I know, my friends, that my words are not what a self-indulgent person likes to hear. There is no easy method and no magic formula by which you can obtain the happiness you all seek. I cannot promise you the precious gifts of heaven on earth and in the spiritual world if you merely do certain prayer exercises. If I were to tell you such things, you would be well justified in being suspicious and doubtful, even though you might undoubtedly prefer to hear this. What I am offering you is real and true. Each one of you has the chance to find out for yourself by trying and by following my advice.
我知道，我的朋友们，我的话不是一个任性的人愿意听到的。想要得到你们都追求的那种快乐是没有容易方法的，也没有什么魔法配方。我不可能仅仅因为你做出一些特定的祈祷练习，就把在地球上和精神世界中找到天堂的珍贵礼物承诺给你的。如果我对你们这样说，你们就很有理由产生怀疑，尽管你们可能毫无疑问地会更愿意听到这样的事。我提供给你们的东西是实际而真实的。你们每个人都有机会通过尝试和追随我的建议来自己验证这一点。

25.17  My advice to begin with is:  Meditate upon the words I have given you here. Consider what the price must be, and what you must expect. Then make your decision. Are you willing to take this path?  Oh, you may say, “It may take a few more lives. I am too tired.”  I can only answer that this again is very shortsighted; if you are tired or weak, it is because your inner forces exhaust themselves in the wrong channels so that your strength cannot organically renew itself as it does in a soul that functions well. If you would only start and not be dismayed by the first struggles, you would finally succeed in setting the inner current right. By doing so, you would set free in yourself a wonderful life force and a spark that will change your life completely.
我作为开始的建议是：对我在这里给你们的话进行冥想。考虑下必须付出的代价是什么，以及你必须预期的情况。然后做出你的决定。你愿意选择这条道路吗？哦，你可能会说，“这也许要花费额外几次生命。我太累了。”我只能回答说，同样，这是非常目光短浅的；如果你很疲劳或虚弱，这是因为你的内部力量在错误渠道中消耗了自己，这使你的力量无法像在一个正常运行的灵魂里那样有机地更新自己。如果你只专注于营造一个开端，而不因为最初的挣扎而灰心丧气的话，你最终会成功地把内部涌流调理得当。通过这样做，你会在自己内心解放出一种美妙的生命力和火花，它们会完全改变你的生命。

25.18  I cannot promise you that all your problems will cease, for they are a necessary part of your path to begin with, a challenge you can learn from if you meet them maturely. However, I can promise you that after you have fulfilled certain fundamental conditions, you will not be depressed any more by your life and your difficulties. I can promise you that your tiredness will cease, and that you will have the strength to go through your difficulties and to bear your cross in the right way, knowing why and what it is all about.
我无法承诺你们说，你们的一切问题都会消失，因为首先来说它们就是你的道路的一个必要部分，一份你能从中学到必要的东西的挑战，前提是你以成熟的态度来面对它们。不过我可以向你们保证，在你满足某些根本性的条件后，你就不会因为自己的生活和困难而沮丧。我可以保证，你的疲劳会消失，而且你会有力量经受自己的困难，并且以正确方式背负自己的负担，同时完全理解其中的原因、整体框架和背景。

25.19  The most difficult thing for you and the most weakening aspect of your life is that you cannot see the reason for anything that happens to you. Only on a path into yourself will you find out the reason, and this alone will give you the strength you need. Furthermore, I can promise you that after a certain time on the path you will enjoy life in spite of your difficulties even before they have actually begun to cease. You will come to relish life in a way you have never been able to do before. I can promise that you will be vibrantly alive—first at intervals, and later more consistently. To the extent you understand yourself and begin to put order into your soul, this vibrant life force will fill you. Life will be beautiful to you in all its reality. So I say to you truly, do not postpone this work for another life. It will not be easier for you then, and you can never get away with not doing the work, for it has to be done. No matter how late you think it is, it is never too late. Whatever you accomplish on this earth will have an eternal value. And when I speak of accomplishment, I do mean the conquering of your lower self.
对你来说最困难的事，以及你生活中最令人产生动摇的方面，是你看不到任何发生在你身上的事情背后的原因。只有在通往自己内心的道路上你才能找到这些原因，而只有这一点能给你带来所需的力量。此外我可以向你保证，在这条道路上前进一段时间后，你可以在自己的困难开始消失之前就能够享受生活。你会以一种之前无法做到的方式欣赏生活。我可以保证你会变得充满生机有活力——一开始断断续续，后来逐渐具有连续性。随着你对自己越加了解，并在灵魂中建立起秩序，这种活跃的生命力量就会以相应的程度注入你。生活中所有现实对你来说都会是美丽的。所以我真诚地对你们说，不要把这份工作推迟到下次生命中再做。那时它也不会变得简单，而你永远无法逃避这份工作，因为这是一定要做的。不论你认为（开始得）有多晚，（实际上）永远都不算晚。你在这个地球上成就的任何事，都有一种永恒价值。当我说“成就”的时候，我指的就是征服你的低级自我。

25.20  Here is another thought, my friends, for this initial decision which you must approach with open eyes:  distinguish the three necessary types of work involved in purifying yourself on this path. One is your outer behavior, the recognition of your apparent faults and qualities, as well as every occurrence that is on the surface. The next phase—and these phases often overlap—is tackling that layer of yourself that does not belong directly to your subconscious, but which you are unaware of because you are deliberately escaping it. This layer has to be treated in a different manner which I will show you. The third and equally important layer is your subconscious mind. Do not believe that what is in the subconscious is so far away that it has no effect on you. You are constantly dominated by your subconscious without knowing it. It is very possible to find out, slowly but surely, what is in your subconscious, at least to some extent.
这里还有一点，我的朋友们，因为你们必须睁大双眼来面对这最初的决定：弄清在这条道路上净化自己时涉及到的三种必要的工作类型。第一，是你的外部行为，认清自己的显著缺点和品质，以及表面上发生的每个细节。下一个阶段——而这三个（类型的）阶段经常重合——是要解决那个不直接属于你潜意识，但因为你故意逃避它而对它没有觉察的层面。这一层面需要以不同的方式进行处理，我会（在后面）展示给你们。第三，也是同样重要的层面是你的潜意识头脑。不要认为存在于潜意识的东西离你很远，而且对你没有影响。你们一直在不知情的情况下受到自己潜意识（头脑）的主宰和支配。要找到你们潜意识中有什么内容（至少到某种程度上）是很有可能的，这个过程缓慢，但必然可以达到。

25.21  Distinguish the trends in you which stand in direct relationship to your conscious will and are thus directly controlled through an act of will. You will also discover trends in you which are connected with your emotions and cannot be directly forced to respond to your wishes. The world of emotion can only change by organic growth, not by pressure and voluntary action, except in an indirect way. Let us assume you find out that deep down you are lacking faith or love. You cannot force yourself to have faith or love, no matter how hard you try directly. But what you can make yourself do is to walk this path, to follow these steps, to overcome perhaps a lack of discipline that makes it so hard for you to work diligently on your path, or tackle whatever else stands in your way quite directly, for instance to work daily on your quota of self-observation and meditation in a detached way. By doing so, you will not work directly on your lack of love or faith, for instance, but you will simply get to know yourself and find out why you lack these attributes. When you gradually understand this without forcing yourself directly to have love or faith, eventually the life force will fill you and will automatically generate these feelings without any direct endeavor on your part. If your emotions begin to change after a few years, you can consider it a wonderful success. The change will happen so naturally that you may not even be fully aware of it at first.
弄清楚自己内心中，哪些趋势与自己的浅表意识力直接相关，进而通过意志力活动就可以对其直接控制。也要弄清楚自己内心中，哪些趋势与你的情绪相关，进而无法直接强迫的它对你的愿望产生应答。情绪世界的改变只有通过有机生长才会发生，而不是（通过施加）压力和自主行为，除非是以某些非直接的方式下。让我们假定，你发现自己内心深处缺乏信念或爱心。你没法强迫自己拥有信念或爱心，不管你多么用力直接尝试（都没用）。但你能做的是使自己走上这条道路，遵循这些步骤，来克服因为缺乏自律导致你很难在自己的道路上勤奋工作的问题，或者解决任何直接阻挡你的其他障碍，比如说每天完成一定配额的自我观察和（超然的）冥想。通过这种做法，你不会直接地调节（比如说）自己缺乏信念或爱的问题，但你会单纯地通过了解自己来发现“为什么自己缺乏这些品质”。当你在不直接强迫自己拥有信念或爱的情况下逐渐理解这一点的时候，这种生命力量最终会灌注到你内部，然后会在不需要你做出任何努力的情况下自动产生这些感觉。如果数年之后你的情绪开始发生变化，你可以把这当做是一个美妙的胜利。这种改变会发生得如此自然以至于一开始你可能都不会清楚地意识到它。

25.22  Study these words now; think about them deeply. Take them to God and ask Him, if you have doubts. He will answer you provided your heart is really open. Believe me, my friends, all this is neither as difficult as it may seem to you now, nor is the path a miracle that will procure happiness for you without demanding from you all you have in honesty, willpower, and effort.
现在学习这些话；深深地思考它们。如果你有疑虑，就把它们带到上帝那里并询问祂。只要你内心真正敞开，祂就会回答你。相信我，我的朋友们，所有这些既不是你们现在看起来的这么难，也不是一个在不要求你给出最大限度的诚实、意志力和努力的情况下为你购得幸福快乐的奇迹。

25.23  I want to say something else to you about this phase of preparation and decision:  expect to have a fight with yourself. It will be the fight between the lower self and the higher self, and your conscious self will determine which side will win. It cannot win without such a fight. It must be a long fight which at first will manifest itself perhaps by preventing you from following this path at all. The lower self may send messages such as,  “I do not believe in it,” or, “It may not be necessary after all,” or “I am too tired,” or “I have no time.”  It is necessary for you to recognize these messages for what they are, and understand where they come from. Use them as a starting point to delve deeper into your soul. Try to see clearly what is really speaking within you when you receive these hidden excuses and pretenses. If you expect this struggle beforehand, you will be able to look and listen and will have a first victory. Also, you will have learned already to some degree the process of uncovering your masks and wrong motives, which will stand you in good stead later on when the lower self will try to obstruct your path by other means. It will simply try to hold on to individual soul-currents. By then you will already know how to deal with it a little better. Do not just put the superficial excuses aside. Test them, deal with them, examine them.
我想对你们说些有关这个准备和决策阶段的其他事情：做好准备和自己打一架。这会是低级自我和高级自我之间的战斗，而你的清醒自我将决定哪边取得胜利。没有这场战斗就没有胜利。这一定会是一次长久的战斗，一开始它的表现形式也许是干脆阻止你迈上这条道路。低级自我可能会发出消息，比如“我不相信这一套，”或“也许这么做并没有必要，”或“我太累了，”或者“我没有时间。”你有必要看清楚这些消息的本质是什么，以及它们来自哪里。把它们作为更深入钻研自己灵魂的起点。当你收到这些隐蔽的借口和伪装时，试着看清楚自己内心是谁在说话。如果你预先对这次战斗有心理预期，你就能（在遇到情况时）看清、听清楚情况，然后获得第一次胜利。另外，你会已经在某种程度上弄懂了“揭露自己的面具和错误动机”的过程，以后低级自我还会尝试用其他手段阻挡你的道路，那时候，这份认识就会使你处于优势地位。低级自我会尝试抓住一些个性化的灵魂涌流。到那时，你就已经知道怎样更好一点地处理它了。不要简单地把这些浅表借口放到一边。测验它们，处理它们，检查它们。

25.24  Many of you are scared of what may come out of your lower self, though you may not be conscious of that fear. It is important to learn to interpret and translate such perhaps vague feelings into concise thoughts. This fear is an important reason why a person shies away from meeting the self.
你们很多人害怕自己低级自我可能会牵引出来的东西，尽管你可能意识不到那份恐惧。学着对这些也许很模糊的感觉进行解读，并把它们翻译成简洁、清楚的思想是很重要的。这种恐惧是一个人回避面对自己的一个重要原因。

25.25  It is childish to imagine that whatever you do not cherish in you does not exist because you avoid facing it. The lower self is immature and ignorant—its very nature is faults and distortions. So I say to you:  Do not shy away from what is in you!  Many people go to psychiatrists and sometimes suffer a breakdown when they come face to face with their lower self in the course of the treatment. This cannot happen to those of you who know that your lower self is not your ultimate being. However, in psychoanalysis the mistaken idea that they are really their lower self can lead people to such self-rejection and self-disgust that they break down. All of you here know that the lower self is merely a temporary layer and does not constitute your entire personality. It is here now to be dealt with, but is not the real you—at least not entirely.
有种想法是“不管自己内心有什么，只要人不去珍惜它，它就不存在，因为你不去面对它，”这是很幼稚的。低级自我既不成熟又无知——它的根本性质就是缺点和扭曲。所以我对你们说：不要躲避存在于自己内心的东西。许多人去看心理医生，有时候他们在治疗的过程中与自己的低级自我进行了近距离接触，从而遭受了崩溃。这种情况不会发生在你们那些知道“低级自我不是自己的终极本质”的人身上。然而，心理分析中这种“人们就是他们的低级自我”的错误概念会给人们带来很大的自我排斥和自我厌恶，最终使他们心理崩溃。在座的所有人都知道了“低级自我”只是一个临时层面，而不等同于你的完整人格。它目前在这里存在着，等着妥善处理，但它不是真正的你——至少不是完整的你。

25.26  Your higher self, which is partly free, already manifests through your good qualities, your generosity, your kindness, or whatever else there is in you that belongs to the higher self. But even where it cannot manifest as yet because it is deeply hidden behind the lower self, your higher self still exists in its shining perfection. How can you reach it unless you penetrate the lower self?  So do not be afraid; do not be shocked when you first encounter your lower self where heretofore you had no idea of it. It is a necessary temporary formation, but it never, never represents the ultimate you. As a matter of fact, reaching the stage where you are shocked about some of its facets that you had not suspected, constitutes a sign of improvement. It strongly implies good progress, for without going through this stage, painful as it may be for a while, you can have no further victory and success. This is part of the path, my friends. If you meditate upon these words and at the same time try to be aware of your fear of your lower self, and your shame of it, and if you learn to live with this truth and knowledge, you will conquer. Then you will meet your fear realistically, and you will not be hiding from it as you are hiding from a few other things in you. This is what I have to say to you tonight.
你的高级自我，一定程度上是自由的，它已经通过你的优秀品质显现出来了，比如你的慷慨，你的仁慈，以及其他类似的品质。然而就算在那些它仍然被低级自我深深遮挡而没法显现的地方，你的高级自我仍然以它闪耀的完美状态存在着。除了穿透低级自我以外，你还能怎么触及到它呢？所以不要害怕；当你在之前对它完全没有概念的地方遇到自己的低级自我时，不要太震惊。它是一个必要的临时现象，但它从不、永远不代表最终的你。事实上，到达一个你对低级自我的一些侧面（之前你没有预料到它们）感到震惊的阶段，这是一个进步的标志。这强烈地表明了很好的进展，因为不经历这个阶段（尽管这可能会让你痛苦一段时间），你就不会有进一步的胜利和成功。这是道路的一部分，我的朋友们。如果你们对这些话进行冥想，同时试着觉察到自己对低级自我的恐惧和你因它而产生的羞耻，并且学着适应这份真相和知识，你就会征服（现阶段的困难）。那时你就能如实地面对自己的恐惧，而且你就不会像躲避自己内心其他一些东西一样躲避它了。这就是今晚我要对你们说的话。

25.27  And now, my dear friends, I am ready for your questions.
现在，我亲爱的朋友们，我准备好回答你们的问题了。

25.28  QUESTION:  Would you mind telling us what the spirits do for recreation?
问题：你介意告诉我们魂灵们怎样娱乐吗？

25.29  ANSWER:  Well, my friends, it is extremely difficult for humans to imagine that spirits live and laugh and enjoy themselves—and work. Spirits in the higher realms do all that, of course, in perfect harmony. What their recreation is depends entirely on their personalities, on their personal tastes, talents, and inclinations. There may be one spirit who is intensely interested in music. This one may live, at least occasionally, in a sphere where it can enjoy this particular pastime. Another may be drawn to art, another to science. Others just enjoys the beauty of Creation, still others will express themselves perhaps in dancing, or in creating certain worlds or parts of worlds, spheres, or certain forms in accordance with their own particular individuality. So there are all forms of recreation in the spirit world. There is the art of interchange through conversation and play. All that you have here is only a very crude copy of what exists in the spirit world.
回答：唔，我的朋友们，要人们想象魂灵们会生活、笑、享受自己——并且工作，这是非常困难的。当然，较高领域中的魂灵们会在完美的和谐状态中做出所有前面提到的行为。它们的娱乐形式完全取决于它们的个性、个人品味、天赋和倾向。可能有一个魂灵对音乐特别感兴趣。它也许会（至少偶尔会）生活在可以享受到这种特定娱乐的位面里。另一个可能被艺术吸引，另一个被科学吸引。其他的只是享受宇宙造物之美，还有其他的也许会通过舞蹈或创造与它们自己的特殊个性相符的特定世界，或者部分世界、位面，或者形状来表达自己。所以精神世界中有各种各样形式的娱乐。也有通过谈话和戏剧进行交流的艺术。你们这里有的一切不过是精神世界中对应物的简陋复制品。

25.30  As a rule, I do not like to talk about this too much, particularly because the intellectual type will not accept it easily. Such a person will say, “Oh, this is childish and primitive.”  But the primitive ideas some people have may be wrong too, because they see or imagine the spirit world in a way which is too human or too unreal. The primitive person who tends toward superstition is not right. Neither are the intellectual people right to deny that anything they consider concrete can exist in spirit, because they accept only what is considered abstract as being part of the spirit world. They forget that concrete and abstract are one in spirit, as everything is one in spirit, at least in the highest spheres. So I do not like to discuss this topic because the right words do not exist to express the spirit world in all the finer shades of its reality, and this may lead to dangerous misunderstandings. I realize that my description will not really give you an adequate picture.
一般来说，我不想过多地谈论这些，特别是因为智力型的人们不会很容易接受它。这样一个人会说，“哦，这很幼稚、很原始。”但有些人的“原始的”想法可能也是错的，因为他们以过于人类或过于不真实的方式看待或想象精神世界。倾向于迷信的原始的人们是不正确的。而智力型的人们否定任何他们认为是有形的、实在的东西能存在于精神世界的做法也是错的，因为他们只接受那种“被认为是抽象的东西才能存在于精神世界”的看法。他们忘记的是有形、实在的和抽象的这两个特征在精神中是一体的，就像所有东西在精神中都是一体的（至少在最高位面中是这样）一样。所以我不想讨论这个话题，因为用来表达精神世界包含的所有更精致微妙侧面的现实的词语是不存在的，而这会引起危险的误解。我意识到我的描述不会真正给你们一个充分的图像。

25.31  QUESTION:  I would like to ask whether the Christ spirit is an all-pervading spirit like God or an individual spirit?
问题：我想问下基督的魂灵是类似上帝那样无处不在的魂灵还是一个个体化的魂灵？

25.32  ANSWER:  It is exactly the same as with God. The substance of Christ is the same as God’s substance; it is all the divine substance. It is the same substance you have in yourself. Whether you call it the divine substance, God’s substance, or Christ’s substance makes no difference. God has given most of this substance to His first creation, the spirit of Jesus Christ. All other beings have received some of this substance and it is up to them to unfold it and enlarge it with the power given to them. Do you understand that?  If you develop yourself, you make your higher self free of the shadows and layers of the lower self. That is the presence you have constantly in you if you can develop it. And this divine spark or higher self is the substance to which we are referring. The presence of God or that of Jesus Christ as a person is something else again. Christ can be felt in person as a presence, in his personification, but that is something altogether different from the divine substance in you. The presence of your own divine substance inside you can only be unfolded by following this very path onto which I am now leading you. To feel the presence of God in His personification—which is almost never the case with a human being, but it is possible for spirits—or to feel the presence of Jesus Christ as a person is an occasional grace that can come unexpectedly, without any knowing or understanding why. These are two entirely different things. Is that clearer now?
回答：这里的情况与上帝的情况完全一样。基督的实质与上帝的实质相同；这些都是神圣实质。它与你自己含有的实质相同。不管你把它叫做神圣实质，上帝实质，还是基督实质没有区别。上帝把最多量的这种实质给了祂的第一个造物，也就是耶稣基督的魂灵。所有其他存有们都收到了一定量的这种实质，而且要靠它们自己，利用拥有的力量把自己包含的这种实质展开并放大。你理解这一点吗？如果你发展自己，你就使自己的高级自我摆脱了低级自我的包围层和阴影。这是始终存在你内部的实质，你只要发展它就可以。而这个“神圣火花”或者“高级自我”就是我们刚才在说的东西。而上帝或耶稣基督作为一个人的出现是另一回事。基督可以在人的层面作为一种存在被感觉到，也就是以他的人格化的形式，但这跟你内部的神圣实质是两码事。你自己的、存在于自己内部的神圣实质只能通过追随这条我现在正在指引给你的道路来展开。而要感觉到上帝以祂的人格化的形式的存在——这种情况在人类的情况下几乎从来是不可能的，但对魂灵们来说是可能的——或者感觉到耶稣基督作为一个人的存在是一种偶尔的恩典，这会在意想不到的、无从知晓的、无法理解的时候发生。这是两种完全不同的事情。现在清楚些了吗？

25.33  QUESTIONER:  Yes.
提问者：是的。

25.34  QUESTION:  I would like to ask something in connection with the fall of the angels. In Isaiah it is said that God created good and evil. Did God create the evil forces and the Luciferic powers too?
问题：我想问下与天使堕落有关的问题。《以赛亚书》中说上帝创造了善良与邪恶。上帝创造了邪恶力量和路西法派系的力量吗？

25.35  ANSWER:  That is a great error, and you will readily understand it now when I remind you of one of the last lectures, The Fall, that explains how this error could have happened. You will remember my explaining that God created the power which He then gave to each of His created spirits. This power could be used in any way depending on the individual spirit’s free will. Now that explains why or how this error could have occurred. It is technically correct to say that God created evil, but it would be more correct to say that God created the possibility for evil if with their free will people—or spirits—use this power against divine law. Is that clear?
回答：这是一个重大错误，我提醒你回顾下我之前解释这个错误是怎样发生的一次讲座，（天使的）堕落，这样你就能明白了。你会想起我给出的解释是：上帝创造了一种力量，然后祂把这种力量给予了祂创造的每个魂灵。这份力量根据个体魂灵的自由意志可以被用于任何方面。这就解释了“这个错误为什么会发生、以怎样的方式发生”的问题。严格来说上帝创造了邪恶是对的，但更准确的说法是上帝以创造“如果人们——或者魂灵们——根据自己的自由意志使用这份力量违反神圣法则”的可能性的方式，创造了（产生）邪恶的可能性。清楚了吗？

25.36  QUESTION:  Yes, but there is always the contrary to each extreme…
问题：是，但每个极端总是有对立面……

25.37  ANSWER:  Certainly the opposite of divine law exists, but that does not mean that God created evil. There is a great difference between creating evil, or giving free will and power to God’s creatures who can then use the power according to divine law or not. I have explained very thoroughly why God gave to all beings the possibility to choose freely. Logically, free will includes the possibility of using it wrongly. If this possibility did not exist, there would be no freedom. And if God’s creatures are supposed to become godlike, they must be free, for freedom is a divine aspect. The wrong use of the power ultimately led to evil through a slow process, even if it did not become evil at the first deviation. I have explained all this very thoroughly. If you read that lecture you will understand. There is no sense in going into this further now because I think it is clear to most of my friends here. Is it unclear to some of you?
回答：神圣法则的对立面当然存在，但这不意味着上帝创造了邪恶。在“创造邪恶”和“把自由意志和力量赋予上帝的生物，于是它们能够选择根据或不根据神圣法则使用这份力量”之间是有重大区别的。我非常详尽地解释了为什么上帝把自由选择的可能性赋予了所有存有们。从逻辑上来说，自由意志包含了错误使用它的可能性。如果这种可能性不存在，就不会有自由存在。如果上帝的生物们要与上帝相似，它们就必须自由，因为自由是一个神圣侧面。对这种力量的错误使用通过一个缓慢的过程最终产生了邪恶，虽然说它没有在一开始偏离的时候就变成邪恶。我已经非常详细地解释了这一点。如果你阅读那次讲座你就会理解。现在继续展开这一点是没有意义的，因为我认为这对在座的多数朋友们都是清楚的。这对你们有些人来说不清楚吗？

25.38  QUESTION:  No, I believe the passage in Isaiah may be a wrong translation—that the Lord created evil. In other words, He gives the possibility for it, but He does not create it.
问题：不，我认为《以赛亚书》中的段落可能是一个错误的翻译——从而说“主创造了邪恶”。换句话说，祂给了（产生）邪恶的可能性，但祂没有创造它。

25.39  ANSWER:  Exactly. You see, the omission of one word very often makes a great deal of difference in the meaning of a sentence. If instead of saying “God created evil,” it had said, “God created the possibility of evil,” the meaning would have been correct.
回答：正是。你看，一个词的省略通常会让一个句子的意思变得非常不同。如果它不说“上帝创造了邪恶，”而是说，“上帝创造了产生邪恶的可能性，”那其中的意思就对了。

25.40  QUESTION:  How is it possible that a person on this path can be so strongly affected by environmental influences?  Can you help me in this respect?
问题：为什么一个走在这条道路上的人可能受到环境影响的如此强烈的作用？你能在这方面帮帮我吗？

25.41  ANSWER:  Well, I can only help you by showing you how to walk the path properly, and I am doing that. The reason why environmental influences are strong can be manifold, since there are different reasons for different people. They are primarily psychological. When the soul is strongly influenced by outer happenings of whatever kind, it is a sign that the soul is not yet free from its own entanglements. If the inner forces are not used according to divine law, the soul is not free. With one person this will manifest in the way the environment affects you, with another it will be different. Whenever the soul is not healthy and mature, certain occasions will bring this out and the soul will produce definite symptoms.
回答：嗯，我只能通过展示给你怎样合适地在这条道路上前进的方式帮助你，而我正在这么做。环境影响很强烈的原因可以是多方面的，因为不同的人有不同的原因。这些原因主要是心理上的。当灵魂受到不论什么种类的外界状况的强烈影响时，这表明这个灵魂还没有摆脱它自己的纠缠。如果内部力量没有根据神圣法则进行使用，那么灵魂就还没有自由。对一个人来说这个问题会以环境对你的产生影响的形式显现出来，对另一个人来说会是另一个情况。每当灵魂还不健康、不成熟，就会有某些场合、时机把这个问题引发出来，而灵魂会产生明确的症状。

25.42  QUESTION:  But why do these symptoms occur, as in my case, when there is high humidity in the air?
问题：但在我的情况里，为什么这些症状会在空气湿度高的时候发生？

25.43  ANSWER:  This happens because everybody’s different odic forces. The odic force is composed differently in each individual and thus responds to different outer challenges. With one person, the cosmic influences affect the soul more strongly; with others, human influences will have a stronger effect, and so on. If you have this particular sensitivity, it is because in your own way there is distortion in your soul which tends, perhaps, to cause sluggishness and takes the first opportunity to make the soul react in this way. The soul is such a complicated mechanism that there are no two souls alike. Certain basic problems are alike, but how all these various trends and currents work, what plays into the lower self or what the lower self will use as an excuse to reinforce a negative trait like sluggishness, say, or where the higher self is diluted by currents of the lower self to constitute a mask or subterfuge, varies with each person. The possibilities are infinite, since no two people react the same way. But the fact remains that this is a sign of a disturbance within the soul. The only remedy is to continue this path to the very end, to follow this course I have outlined here. If you were entirely purified and healthy, you would not be sensitive to weather or anything else that might come your way from outside.
回答：这种情况是因为每个人都有不同的自然力。每个个体的自然力组成不同，因此对外界刺激的应对方式也不同。对一个人来说，宇宙影响的作用对灵魂的效果更强；对其他人来说，人类影响力的作用更强，如此等等。如果你有这种特殊的敏感性，这是因为以你独特的方式，你的灵魂中存在扭曲，它可能会引发惰性并且一有机会就会让灵魂做出这种反应。灵魂的机理是如此复杂，以至于没有哪两个灵魂是相似的。某些基本问题是相似的，但所有这些不同的趋势和涌流怎样运作，哪些因素会让低级自我占便宜，哪些因素会被低级自我作为借口来增强某个负面特性（比如惰性），高级自我在哪些地方被低级自我的涌流稀释了从而产生了面具或欺骗手段，这些问题在每个人身上都是不同的。其中的可能性是无限的，因为没有哪两个人反应方式完全一样。但事实仍然是：这是一个灵魂中存在扰动的标志。唯一的补救办法是把这条道路进行到底，追随我在这里规划出的行动方向。如果你真的完全纯净而且健康，你就不会对天气或任何可能从外部找到你的东西产生敏感。

25.44  QUESTION:  Then I would not be here, I suppose.
问题：那样的话我就不会在这里（地球上）了，我猜。

25.45  ANSWER:  That is right.
回答：没错。

25.46  QUESTION:  Can you tell me if there is any spiritual significance or reason for the different races?
问题：你能否告诉我（存在）不同种族这种现象背后有没有任何精神方面的重要性或原因？

25.47  ANSWER:  Oh yes, indeed. There is no such thing as coincidence. I should like to state briefly that human beings have to go through karmas. For instance, if a person is born into a certain race in which there is much collective suffering, it is of course due to this particular soul’s karma. The more unified humanity becomes through spiritual development, the more will different races cease to exist. You can begin to see even at this time the slow but sure development in the direction of eliminating racial differences. In a few thousand years from now, perhaps in about fifteen hundred years, the effect of this development will be very noticeable; nations, religions, and races will have few of their differences left. This will be a sign of unification and spiritual improvement. However, until such time, there will be and must be differences, for only by encountering obstacles can you grow. This holds true for the individual’s life, and it certainly applies also to the development of groups.
回答：哦是的，确实有。没有所谓的“巧合”这回事。我想简略地说：人类们需要经历因果/业报。比如说，如果一个人出生在一个充满集体苦难的种族中，这当然是因为这个特定灵魂的因果/业债。人类通过精神发展变得越加统一，不同的种族就会越加停止存在。就在目前，你们已经可以开始看到在这个消除种族区别的方向上产生的缓慢但确切的发展。在今后的几千年时间里，也许大约一千五百年里，这种发展的效果会变得非常显著；国家/民族，宗教和种族之间就不会有什么区别了。这会是统一和精神发展的标志。但是，在此之前，会有而且必须有区别和不同，因为只有通过遇到障碍你们才能成长。这个道理对个体生命来说成立，它对不同群体的发展当然成立。

25.48  You may wonder why certain races do not suffer from being different from others. Then you might ask, “What can they learn from that?”  Indeed, they, too, have something to learn. Perhaps they need to learn the responsibility that arises from being spared sufferings that other peoples may have. Besides, this angle is not the only one to be considered. A spirit can be born into a race or nation because spiritually, emotionally, characterwise and psychologically it belongs to this group and therefore has the best opportunity to unfold there. Differences will exist as long as disunity exists on earth and humanity has not learned to overcome it. As any difficulty or apparent disadvantage can be a cure, which it must be if the person is on the right path, advantage can be a cure, too. Through differences of race, religion, nationality, or various other categories, humanity can become stronger and advance faster in spiritual development precisely because frictions exist. Without friction, development can never proceed. It is only a question of how the difficulty is met, always; how is it met individually and collectively. Is that clear?
你可能在想为什么某些种族不会因为与别的种族不同而受苦。这时你可能会问，“他们能从中学到什么？”而他们，确实也能从中学到某些东西。也许他们需要学会承担起从“免于遭到其他种族的人们遭受的痛苦”这个情况中产生的责任。另外，这个角度不是唯一要考虑的。一个魂灵可以因为“它在精神方面、情绪方面、个性方面和心里方面属于这个群体因而在这里有最好的机会展开自己”而出生在某个种族和民族中。只要地球上还存在不统一、人类全体还没有学会克服它，就始终会有区别和不同点存在。就像任何困难或明显劣势可能是一种解药（如果这个人在正确道路上的话它必然是这样）一样，优势也可能是一种解药。通过不同的种族、宗教、国籍和其他各种不同，人类全体能够变得更强，能在精神发展方面进展得更快，这正是因为摩擦的存在。没有摩擦，发展永远无法进行。唯一的问题是困难是怎样被面对的，总是这样；它是怎样在个体层面和集体层面被面对的。清楚了吗？

25.49  QUESTIONER:  Yes, thank you.
提问者：是的，谢谢你。

25.50  QUESTION:  But is not the variety of different races and their trends part of the beauty of life?
问题：但是不同种族的多样性和他们的不同趋势难道不是生命的美丽的一部分吗？

25.51  ANSWER:  Yes, the variety is fine, but variety on your earth means friction and hostility. In spirit there is infinite variety in everything, but it is unlike that on your earth where you have what you call “better races” and “inferior races.”  That, of course is karmic for both groups and not only for the groups which are sometimes persecuted. That is also a thought upon which you can now meditate.
回答：是的，多样性是好的，但多样性在你们的地球上意味着摩擦和敌意。在精神（世界）中所有东西都有无限的多样性，但那不像你们地球上的情况，这里你们有所谓的“优等种族”和“劣等种族。”这种情况当然是有因果/业报的原因，这些原因涉及到双方两个群体而不仅仅针对那些有时受到迫害的群体。这也是一个你们可以进行冥想的话题。

25.52  QUESTION:  Is there something wrong if, working on this path of development, you find that you have one particular fault and you are trying to overcome it through meditation and prayer and yet you don’t seem to get further along?
问题：如果在这条道路上工作时，你发现自己有某个特定的缺点，然后试图通过冥想或祈祷克服它，但是似乎没有进展，这种情况下是不是有什么不对？

25.53  ANSWER:  I will not say that there is something wrong, but I would say that some method of some key that you need is lacking. You try hard, again and again, in one particular direction perhaps too forcefully. Your lower self resists that. Maybe I can help you by saying this:  My dear friend, you must realize one thing. When I spoke about the battle and said, fight, I meant just that. It takes the lower self a very long time to become ready to give up its faults, its resentments, its sluggish ways. You must realize that this lower self lives in you strongly, even though you are not aware of it consciously and do not see its significance, because consciously you only know your good will to work in the right direction. This realization is the first essential step. Without making this realization, you can get nowhere, no matter how good the will of your conscious being is. What most people do not accept, at least not until a pretty distant point is reached on this path, is the discrepancy in themselves; how one part can want one thing, while another part wants exactly the opposite. This conflict of opposite desires, which at first is always subconscious, must be taken out and brought into the light of consciousness. Without this essential step there can be no further success.
回答：我不会说这里有什么不对，但我会说你需要的某种方法的关键缺失了。你一次又一次地用力尝试，在那个特定方向上可能太强硬了。你的低级自我会抵抗这一点。也许我这样说可以帮你：我亲爱的朋友，你必须意识到一件事。当我谈到（与低级自我的）战争时，我说（你们）要进行战斗，我的意思正是字面意思。低级自我要花很长时间才能愿意放弃它的缺点、它的不满和它懒散的做事方式。你必须意识到：虽然你没有清醒意识到它的存在，也看不到它的重要性，因为在有意识层面你只知道以自己的良好意图在正确的方向上工作，（但实际上）这个低级自我在你内心是很强大的。这种意识是第一个关键步骤。没有这个意识，不论你的自主意识有多好的意图，你都不会有进展。多数人（至少在到达这条道路上很远的一个节点以前）不愿意接受的，是自己内部的矛盾；（自己内心的）一部分想要一件事，而另一部分想要截然相反的事。这种相反欲望的冲突（一开始总是潜意识的），必须被梳理出来并带入显意识的光明之下。没有这个关键步骤就不会有进一步的成功。

25.54  Therefore, my advice is to not try for the moment to force yourself to feel what you have been so far incapable of feeling. Instead, use your efforts in the direction of making that part of you conscious that is responsible for your lack of success so far. Relax completely, not only physically but also emotionally. Then try to let your lower self come to the surface, so that you can actually become aware of it. When your enemy becomes visible, you can fight successfully:  as long as your enemy is invisible, you cannot succeed. So do not be afraid to let out what is in you. Very quietly say to yourself and, in your meditations, to God:  “This is the way I am now. At least a part of me is this way, the part I do not like or cherish. I want to accept this aspect as part of my being, knowing that I cannot change anything without doing so freely. I realize that what will manifest is not my whole being. But there are two contradictory forces in me, and I have to become aware of both, including the one I have so far ignored. In order to become what I would like to be, I must first, without fear or shame and vanity, face what is in me.”
因此，我的建议是暂时先不要在强迫自己去感受目前为止感受不到的东西。相反，把你的努力用在让自己意识到目前为止导致你缺乏成功的那个部分。完全彻底地进行放松，不仅是身体上而且是情绪上。然后试着让你的低级自我来到表面，使自己能够实际觉察到它。当你的敌人变得可见时，你才能成功的战斗：只要你的敌人是隐形的，你就无法成功。不要害怕释放出自己内部的东西。非常安静地对自己（如果在冥想中就对上帝）说：“这是我目前的状况。至少我的一部分是这样的，那个我不喜欢、不珍惜的部分。我要接受这一方面作为我存在的一部分，知道如果不主动这么做，我就没法改变任何事。我意识到将会显现的东西不是我的完整存在。而是有两种矛盾的力量存在于我内部，而我必须对两者都有觉察，包括目前为止我忽视的这个。要变成我希望自己成为的样子，我必须首先，在没有恐惧或羞耻感或虚荣的情况下，面对自己内部的真实情况。”

25.55  Ask God to help you to achieve that end, and then let the lower self come out. Listen to your feelings in connection with incidents that have always brought out this trend in you. Then try to translate these feelings into concise thoughts and words. Do this constantly and you will be successful in this respect. You will then come to the point when you can quietly realize these two contradictory currents in you:  the one in which you see your imperfection, and the opposite one, where you know in theory how you would like to feel and react, even though you cannot as yet do so. First compare these two currents and learn for the time being to accept your present imperfection in comparison with the perfection you know to be right. Accept this imperfection with humility. If you do that constantly, your feelings will change.
请求上帝帮助你达到那个目标，然后把低级自我释放出来。倾听自己的感觉，把它们跟那些总是把这种趋势激发出来的事件联系起来。然后试着把这些感觉翻译成简洁的思想和语言。坚持这么做，然后你就会在这方面取得成功。你会到达一个节点，从而可以安静地识别出自己内心中这两种互相矛盾的涌流：其中一种涌流中你看到自己的不完美之处；在相反的一种涌流中，你从理论上知道自己想要有怎样的感觉和做出怎样的反应，尽管目前你还做不到。首先比较这两种涌流，暂且学会接受代表着自己临时的不完美的这个涌流。以谦逊的态度接受这份不完美。如果你坚持这样做，你的感觉就会发生变化。

25.56  Constant self-observation and self-honesty will have a surprising effect on you. By following this procedure, you will learn to follow the law of reality, accepting yourself as you are and thus learning true humility. This awareness will generate a new strength in you even before you have attained perfection in this respect. After a period of this kind of training, very gradually your feelings will begin to change. This is the only way, my dear.
持续的自我观察和自我诚实会在你身上产生令人惊奇的效果。通过坚持进行这个过程，你将学会遵循“现实法则”，实事求是地接受自己并因此学到真正的谦逊。这种觉察甚至会在你在这方面达到完美状态之前，就从你内部产生一种新的力量。经过一段时间的这种训练，你的感觉和情绪会逐渐地发生变化。这是唯一的方式，我亲爱的。

25.57  I meant to talk about this topic in a following lecture, but since your question came up, it is good for all of you that I answered it now. I will come back to this because it constitutes the method of purification of emotions not influenced by an act of direct willpower. This cannot be emphasized enough. It often takes some time until this method is clearly understood, but once it is, it cannot fail to bring results. Some of you may get bored hearing this again and again, especially those of my friends who have heard this from me in their private sessions, but I cannot say it often enough. Many of you have understood this process only in your brain, but your inner self has not yet perceived or understood it entirely. Do you understand?
我本来打算在之后的某个讲座中谈这个话题，但考虑到你提到这个问题，我现在回答这一点对你们所有人都有好处。我（以后）会回来继续讨论这个问题，因为它是“在不受直接意志力影响的情绪领域进行净化”的方法的一部分。这个问题怎么强调都不为过。要清楚理解这个方法通常会花很长时间，但一旦它被清楚理解，就不可能不产生（有利的）结果。你们有些人（以后）可能会对一次又一次听到这个问题感到无聊，特别是那些在私人会议中从我这里听到这个内容的朋友们，然而，尽管这样也仍然不够。你们许多人只在大脑里理解了这个过程，但你们的内部自我还没有完全觉察到或理解到它。你理解了吗？

25.58  QUESTION:  Yes. How do you get your inner self to understand the emotional self?
问题：是的。你怎样才能让自己的内部自我理解情绪自我？

25.59  ANSWER:  Do not try to make it understand to begin with. Let it out first. That is the first half of the process. You cannot educate or reeducate your emotional self before it is completely on the surface. To be in a hurry to change these emotions is not only an impossible goal and therefore your hurry is futile and unrealistic, but this period of constant self-recognition where you make comparisons between your emotions which conflict with what you have long recognized as being right teaches you humility. To accept yourself as an imperfect being is accepting yourself in the right way, without guilt feelings and self-abuse. The latter are unhealthy and unproductive. You must learn to take yourself realistically and say, “In so many respects, I am still imperfect. In so many others, I have reached perfection. If in the work on this path I should find out that I am less perfect than I thought, I will accept this too, in order to be able to change it.”  Realize that before your imperfections can change, you have so many other things to learn from this very imperfection in you.
回答：第一步是不要试图让它理解。首先，允许它表现出来。这是整个过程的前半段。你没法在自己的情绪自我完全处于意识表面之前对它进行教育或再教育。在匆忙的状态中改变这些情绪不仅是一个不可能的目标（因此你的匆忙是无用的、不实际的），而且这是一段持续的自我认知的时期，（在此期间）你把自己内心中有冲突的情绪与那些你一直认为是正确的情绪进行对比，它会使你懂得谦逊。要接受“自己（暂时）是一个不完美的存在”，意味着正确地接受自己，不抱有愧疚感和自虐（倾向）。后两者是不健康、没有建设性的。你必须学会实事求是地应对自己，并且说，“在很多方面，我仍然不完美。在很多其他方面，我已经达到完美。如果在这条道路上的工作中，我发现自己没有我想象的那么完美，我也会接受这一点，从而有能力改变这一点。”要意识到在你的不完美之处能发生变化前，你需要从这个不完美之处中学到很多其他东西。

25.60  Learn the proper attitude with which to meet your lower self, accept your inability to change something by a mere effort of will. All this is necessary on your path, and therefore the still existing imperfections fulfill a definite purpose. This does not mean that you should nurse them, justify their existence, or, worse, become lazy and not do anything about them. This would be the wrong opposite extreme. To find the right middle path in everything is part of the path. Learn to accept yourself without fear, without shame, and without pride. Where you still discover these trends, examine what is behind them and gain further understanding about yourself. If you follow this system for a while and ask at the same time for God’s grace to help you, you will get further and further, although you will not change these currents immediately. You will grow in self-recognition and self-understanding, however. This is fundamental. The proper spirit of accepting the lower self with the necessary humility is of much greater importance than any of you realize as yet. You mistakenly want to skip this phase altogether, and by so doing you will inevitably become discouraged eventually because your endeavors will always wind up in a dead-end street.
找到面对自己低级自我的合理态度，接受“自己无法仅仅通过一次意志里的努力就改变某些事情”的事实。所有这些都是你们在道路上的必要过程，因此那些仍然存在的不完美之处起到了一个明确的目的。这不是说你应该培育这些不完美之处，把它们的存在合理化，或者（更糟的情况是）变得懒惰而不采取任何措施。这就是错误的对立极端。在一切事情中找到正确的中间路径，这是这条道路的一部分。学着不带恐惧、不带羞耻、不带骄傲地接受你自己。在那些你仍然会发现这些趋势的地方，检查它们背后的原因，并从中加深对自我的理解。如果你在这个系统下坚持一段时间，同时请求上帝的恩典帮助你，你就能走得越来越远，虽然说你不会立刻改变这些趋势。但是你会在自我认知和自我理解方面得到成长。这是根本性的。合理的接受低级自我的态度以及必要的谦逊，其中的重要性比你们任何人意识到的还要重大很多。你们错误地想要完全跳过这个阶段，但这样的话你最终会不可避免地变得灰心丧气，因为你的努力总是会进入一个死胡同。

25.61  And now I will retire, my dear friends. God’s blessings go to all of my dear friends far away and to all of my dear friends in this room, as well as to those who have found their way to us for the first time tonight. May this be a turning point in their lives. Peace be with you; be in God!
现在我要退回了，我亲爱的朋友们。上帝的祝福会流向所有远处的亲爱的朋友们，以及所有这间屋子里的朋友们，也会流向今晚第一次来到我们这的朋友们那里。愿这（次讲座）成为他们生活中的一个转折点。愿你们得到平静；与上帝同在！
_
_
©2019 International Pathwork Foundation
Translated by Mark.  Edited by April.
微信订阅号 @Pathwork Lectures 心路讲座中文
